src/MonoMod.Core/Platforms/Systems/SystemVABI.cs

```diff
@@ -30,6 +30,32 @@ public static TypeClassification ClassifyAMD64(Type type, bool isReturn)
             return TypeClassification.InRegister;
         }
 
+        public static TypeClassification ClassifyARM64(Type type, bool isReturn)
+        {
+            // ARM64 uses a byref ret buf in x8 if the return size is > 16, but there is no way to model this
+            // currently so just return InRegister for now so the return buffer fixup is effectively disabled
```

Comment:
Out-of-date comment?

Reply:
How? It looks like this case should be handled, or at least handleable. What is actually stopping us from implementing this properly?